utils/test_fv3gfs_dataset.py

```diff
@@ -36,15 +48,18 @@ def __contains__(self, key):
         return key in self.items
 
 
-def test_FV3GFSDataset_init():
-    dataset = FV3GFSDataset(DotDict(TEST_PARAMS), TEST_PATH, True)
-    dataset.in_names == ["UGRD10m", "VGRD10m", "PRMSL", "TCWV"]
-    dataset.out_names == ["UGRD10m", "VGRD10m", "TMP850", "TCWV", "Z500"]
+@pytest.mark.parametrize("params", [TEST_PARAMS, TEST_PARAMS_SPECIFY_BY_NAME])
+def test_FV3GFSDataset_init(params):
+    dataset = FV3GFSDataset(DotDict(params), TEST_PATH, True)
+    assert dataset.in_names == TEST_IN_NAMES
+    assert dataset.out_names == TEST_OUT_NAMES
 
 
 def test_FV3GFSDataset_len():
     dataset = FV3GFSDataset(DotDict(TEST_PARAMS), TEST_PATH, True)
-    assert len(dataset) == 1459
+    full_path = os.path.join(TEST_PATH, "*.nc")
+    expected_length = len(netCDF4.MFDataset(full_path).variables["time"][:]) - 1
```

Comment:
Why minus one?

Reply:
Because the training requires paired samples at (t, t+1), so we don't have an output for the last timestep. There's a comment to this effect in `FV3GFSDataset`.